packages/web/src/utils/menus/utils.ts

```diff
@@ -0,0 +1,187 @@
+/**
+* Handle event for keyboard control of menu
+* @param {Event} event
+* @param {Element} menu
+* @param {any[]} queue
+*/
+export async function handleKeyDownMenu(event, menu, queue) {
+  let key = event.keyCode || event.which;
+  let currentIndex = queue.indexOf(document.activeElement);
+  let activeElement = queue[currentIndex];
+
+  // Down arrow
+  if (key == 40) {
+    event.preventDefault();
```

Comment:
Instead of adding `event.preventDefault()` for each key, could we reduce the code and prevent default for all key events or is that going to cause any accessibility issues?

Reply:
That makes sense, I'll resolve this comment!